### ISSUE TITLE
Сообщение о том, что другой персонаж посмотрел на игрока

### ISSUE THREAD

**Issue by vehsamrak** [MagicGreenHat/Kingdom#52]

**Comment by Muderru** [comment]
Мне кажется, что такие сообщения нужны только в играх с фри-пк и фулл-лутом, чтобы предупреждать игрока, что его могут раздеть. Если в игре пк нет, и игрок свой стаф потерять не может, то такая информация для него будет бесполезным спамом.

**Comment by vehsamrak** [comment]
@Pacifist03 согласен. Я думаю сделать это включаемым параметром (по дефолту отключенным) в рамках задачи #61
